ui/app/AppLayouts/HomePage/HomePage.qml

```diff
@@ -29,9 +29,6 @@ Control {
 
     property bool useNewDockIcons: true
 
-    required property bool hasUnseenACNotifications
-    required property int aCNotificationCount
```

Comment:
Why did you remove it from here? I believe it needs to stay for the HomePage, it's a separate section on its own and we don't have the main left navbar

Reply:
Well, now the notifications are accessible in the `Dock bar` so probably it's ok this way? 

<img width="797" height="195" alt="Screenshot 2025-09-04 at 12 20 52" src="https://github.com/user-attachments/assets/84ffb2eb-19aa-43bd-82c0-2d94dbc3297c" />

Reply:
Hmmm interesting... what does clicking it do?

Reply:
It moves to the Activity Center section!

Reply:
> It moves to the Activity Center section!

Ok, I see... fine for me :)